### DISCUSSION TITLE
How to specify fractional values for Tasmota thresholds (eg.  CurrentHigh)?

### DISCUSSION THREAD

**Discussion by DurvalMenezes** [arendst/Tasmota#18484] [Q&A] [answered]
As per the subject, if I enter at the console: 
```
CurrentHigh 3.4
```
It responds with: 
```
17:20:34.551 MQT: stat/tasmota_gen03/RESULT = {"CurrentHigh":3}
```

Tried other combinations (eg, comma instead of a dot) to no avail, the fractional part is always ignored. 

So, is there a way to set a threshold to a fractional (non-integer) value, and if so, how? 

TIA!

**Comment by barbudor** [comment, marked as answer]
Considering that the unit is milliamp, do you really need a fractional part

**Reply by DurvalMenezes** [reply]
Thanks! I was assuming the value for `CurrentHigh` specially  was in  amps as in the `Status 8` output, should have read the FM better... 🤦‍♂️
